bin/bootstrax

```diff
@@ -250,13 +248,16 @@ max_timetamp_diff = 5
 
 # The maximum number of retries for processing a run. After this many times of retrying
 # to process a run, the DAQ-group has to either manually fix this run or manually fail it.
-max_n_retry = 20
+max_n_retry = 10
+
+# Check that there are runs that are waiting to be processed. If there are few,
+# an eventbuilder should not process new data.
+max_queue_new_runs = 2
 
-# Bootstrax retries runs multiple times. If there have been this many fails we could lower
-# the resources to be somewhat more lenient on the CPU and RAM of the eventbuilders. Use
-# this option with care as we might be running in a sub-optimal mode that may not be
-# noticed and eventbuilders may be spending more time on trying to reprocess failed runs.
-lower_resources_after_n_failures = 10
+# Remove any targets or post processing targets after the run failed this
+# many times.
+# NB: string match so events applies e.g. to event_basics, peak to e.g. peaklets
+remove_target_after_fails = dict(events=2, hitlets=2, peak=3, online=4, ALL=5)
```

Comment:
Depends on what is causing the issue, not all fails are always related to data quality.

I want to avoid failing to process medium level stuff due to one of the highest level plugins running onto some issue.

Reply:
At some point we'll figure out how to extract this information from the stack trace, and then we'll know exactly what should get deleted.

Reply:
This would be great but perhaps not during commissioning.